datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/license_headers.py

```diff
@@ -0,0 +1,52 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pathlib
+
+import click
+
+from ...constants import get_root
+from ...license_headers import validate_license_headers
+from ...testing import process_checks_option
+from ...utils import complete_valid_checks
+from ..console import CONTEXT_SETTINGS, abort, echo_failure, echo_info, echo_success
+
+IGNORES = {"datadog_checks_dev": ["datadog_checks/dev/tooling/templates"]}
+
+
+@click.command(context_settings=CONTEXT_SETTINGS, short_help='Validate license headers in python files')
+@click.argument('check', shell_complete=complete_valid_checks, required=False)
+@click.pass_context
+def license_headers(ctx, check):
```

Comment:
I think we should have a sync flag that users can run to automatically add license headers if we add this to CI

Reply:
Sounds good, I can do a follow up with `--fix`.